lib/logging/logging.dart

```diff
@@ -331,11 +343,24 @@ class LogDetailsUI extends CoreElement {
   LogData get data => _data;
 
   set data(LogData value) {
+    // TODO(devoncarew): Reset the vertical scroll value if any.
+
     _data = value;
 
     if (_data != null) {
-      // TODO(dantup): Can we format the JSON better?
-      message.text = _data.message;
+      final String str = _data.message;
```

Comment:
nit: There are a usages of both `_data.message`  and `str` below here, maybe they should be the same?

Reply:
Yes - good catch; cleanup this up a bit.